frontend/views/components/StartInviting.vue

```diff
@@ -0,0 +1,66 @@
+<template lang="pug">
+section.card.c-card
+  svg.c-svg
+    use(xlink:href='#svg-conversation')
+  div
+    i18n.title(tag="h3") Invite member to your group
+    p Start sharing your resources with the ones who matter to you!
+    .buttons.is-start
+      i18n(
+        tag="button"
+        @click="openModal"
+      ) Add members
+      router-link.button(
+        to="/invite"
+        data-test='inviteButton'
+      ) {{L('Invite members')}}
```

Comment:
Can we have consistent spacing here? I think it's more readable as:

```js
{{ L('Invite members') }}
```

It might also make it easier to write the parser if it's done this way...

Reply:
I'll change it now and I'll work next on improving our linting rules so we avoid this type of picky comments in the future.